README.md

```diff
@@ -7,22 +7,33 @@ Command-line application to generate a report for any VCF (Variant Call Format)
 
 ## Usage
 ```
-java -jar vcf-report.jar -i <arg> [-o <arg>] [-f] [-t <arg>] [-d]
- -i,--input <arg>      Input VCF file (.vcf or .vcf.gz).
- -o,--output <arg>     Output report file (.html).
- -f,--force            Override the output file if it already exists.
- -t,--template <arg>   Report template file (.html).
- -d,--debug            Enable debug mode (additional logging and pretty printed report.
-```
-```
-java -jar vcf-report.jar -v
+java -jar vcf-report.jar -i <arg> [-o <arg>] [-f]
+       [-t <arg>] [-pd <arg>] [-ph <arg>] [-d]
+ -i,--input <arg>         Input VCF file (.vcf or .vcf.gz).
+ -o,--output <arg>        Output report file (.html).
+ -f,--force               Override the output file if it already exists.
+ -t,--template <arg>      Report template file (.html).
+ -pd,--pedigree <arg>     Comma separated list of pedigree files (.ped).
+ -ph,--phenotypes <arg>   Semicolon separated list of CURIE* formatted
```

Comment:
Proposal:
```
Comma-separated list of phenotypes per sample (e.g. HPO:123 or HPO:123;HPO:234 or sample0/HPO:123,sample1/HPO:234). Phenotypes are CURIE formatted (prefix:reference) and separated by a semicolon.
```

Reply:
not sure, doen't that imply that de phenotypes are comma separated instead of the sample/pheno combinations?

Reply:
How about: ```Comma-separated list of sample-phenotypes```?